real_preprocessing/src/corner_detection.cpp

```diff
@@ -0,0 +1,88 @@
+#include "ros/ros.h"
+#include "apriltag_ros/AprilTagDetectionArray.h"
+#include "sstream"
```

Comment:
I dont think you are using this library. If you aren't then remove it.

Reply:
Not needed so removed*